app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,42 +1,16 @@
-import React, { useState } from "react";
+import React from "react";
 
 import { Alert } from "neetoui";
 
-import notesApi from "apis/notes";
-
-const DeleteAlert = ({
-  refetch,
-  onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
-}) => {
-  const [deleting, setDeleting] = useState(false);
-
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
-      onClose();
-      setSelectedNoteIds([]);
-      refetch();
-    } catch (error) {
-      logger.error(error);
-      setDeleting(false);
-    }
-  };
-
-  return (
-    <Alert
-      isOpen
-      isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
-      onClose={onClose}
-      onSubmit={handleDelete}
-    />
-  );
-};
+const DeleteAlert = ({ onClose, onDelete, isOpen }) => (
+  <Alert
+    isOpen={isOpen}
+    message="Are you sure you want to delete the note? This action cannot be undone."
+    size="large"
+    title={`Delete Note`}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Rendering%20strings%20and%20blank%20spaces%20in%20JSX

Reply:
This is resolved.

Updated title prop as follows

`title="Delete Note"`